ttnn/cpp/ttnn/operations/examples/example_multiple_return/device/single_core_program_factory.cpp

```diff
@@ -15,18 +15,18 @@ ExampleMultipleReturnDeviceOperation::SingleCore::cached_program_t ExampleMultip
 
     const auto& input_tensor = tensor_args.input_tensor;
 
-    auto output_tensor1 = tensor_return_value.at(0).value();
-    auto output_tensor2 = tensor_return_value.at(1).value();
+    auto output_tensor1 = tensor_return_value.at(0);
+    auto output_tensor2 = tensor_return_value.at(1);
 
     auto src_buffer = input_tensor.buffer();
-    auto dst_buffer1 = output_tensor1.buffer();
-    auto dst_buffer2 = output_tensor2.buffer();
 
     tt::tt_metal::Program program{};
 
     tt::DataFormat cb_data_format = tt::tt_metal::datatype_to_dataformat_converter(input_tensor.get_dtype());
     uint32_t single_tile_size = tt::tt_metal::detail::TileSize(cb_data_format);
-    tt::DataFormat cb_data_format_output = tt::tt_metal::datatype_to_dataformat_converter(output_tensor1.get_dtype());
+
+    auto output_dtype = output_tensor1.has_value() ? output_tensor1.value().get_dtype() : output_tensor2.value().get_dtype();
```

Comment:
This part of the example is a little weird. I don’t require the change, but just want to call it out, this is far from how real code will look like.

Reply:
Since the output1 tensor can be nullopt, I made it so that in such cases, the dtype is retrieved from output2. Is this modification unnecessary?

Reply:
I don’t mind. Nevermind.